pkg/csi/service/common/vsphereutil.go

```diff
@@ -897,6 +895,8 @@ func isExpansionRequired(ctx context.Context, volumeID string, requestedSize int
 		return false, err
 	}
 
+	log.Infof("isExpansionRequired: Found current size of volumeID %q to be %d Mb while "+
```

Comment:
Verbiage gives the impression that requested size is always less ?
Can we re-phrase this

Reply:
Done